storage/rocksdb/rdb_datadic.h

```diff
@@ -112,6 +112,7 @@ const char RDB_UNPACK_DATA_TAG = 0x02;
 const size_t RDB_UNPACK_DATA_LEN_SIZE = sizeof(uint16_t);
 const size_t RDB_UNPACK_HEADER_SIZE =
     sizeof(RDB_UNPACK_DATA_TAG) + RDB_UNPACK_DATA_LEN_SIZE;
+const size_t RDB_COVERED_BITMAP_SIZE = sizeof(uint16_t);
```

Comment:
Also, since your code updates the sk versions, could you provide test cases that verify being able to read/write both formats? You may need to set debug points (see other tests that do this) in the code to allow you to execute as if the index was of one format version or the other, despite the actual latest index format version to be used.